annotate.R

```diff
@@ -24,21 +24,15 @@ suppressWarnings(suppressMessages(library(readr)))
 opts <- docopt(doc)
 
 batch_id <- opts[["batch_id"]]
-
 external_metadata <- opts[["external_metadata"]]
-
 cell_id <- opts[["cell_id"]]
-
 format_broad_cmap <- opts[["format_broad_cmap"]]
-
 plate_id <- opts[["plate_id"]]
-
 perturbation_mode <- opts[["perturbation_mode"]]
-
 metadata_dir <- paste("../..", "metadata", batch_id, sep = "/")
+backend_dir <- paste("../..", "backend", batch_id, unlist(strsplit(plate_id, "_"))[1], sep = "/")
```

Comment:
@gwaygenomics Do you know why the string is being split?

Reply:
As we discussed on the brief chat, indeed, it looks like I am splitting the plate ID because I am inputing the plate ID with the `sc_type` tagged ([here](https://github.com/broadinstitute/cmQTL/blob/master/0.pilot-determine-conditions/generate-sc-profiles.sh#L78)).

This is a result of the output from the modified `aggregate.R` call above, based on this [line](https://github.com/broadinstitute/cmQTL/blob/master/0.pilot-determine-conditions/generate-sc-profiles.sh#L62)

This will definitely break in situations where the plate ID has an underscore.

Will need to address!